src/requests/search.request.ts

```diff
@@ -69,4 +71,78 @@ export class SearchRequest {
       params: queryParams,
     });
   }
+
+  searchSalesNavigator({
+    skip = 0,
+    limit = 25,
+    filters = {}
+  }: {
+    skip?: number;
+    limit?: number;
+    filters?: PeopleSalesNavigatorSearchFilters;
+  }): Promise<GetSalesNavigatorPeopleSearchResponse> {
+    const graphqlQuery = (() => {
+      const graphqlFilters = [
```

Comment:
Can we somehow leverage the `paramsSerializer` util parse these parameters?
I think it's the same logic.